code/script.js

```diff
@@ -1,19 +1,121 @@
 // Start here
 
 // Step 1 - Welcome and introduction
-// Your code goes here
-alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
-
+alert(`Welcome to our Yia's Pizzeria. Ready to Start? - Click 'OK' to begin.`);
+let firstName = prompt("What is your name?");
+// alert("Hi " + firstName + "!");
+alert(`Hello ${firstName}!`);
+console.log(firstName);
```

Comment:
Please remove all the console.log() around the code. Generally, they need to be deleted when the code has been tested and works as meant to 👍